pkg/kubectl/stop_test.go

```diff
@@ -558,52 +558,43 @@ func (c *noSuchPod) Get(name string, options metav1.GetOptions) (*api.Pod, error
 	return nil, fmt.Errorf("%s does not exist", name)
 }
 
-type noDeleteService struct {
-	coreclient.ServiceInterface
+type noDeletePod struct {
```

Comment:
ReaperFor will return nil for service now (since there is no explicit Service reaper). Hence updated the test to use Kind Pod instead of Kind Service.